packages/assistify-ai/server/methods/SmartiWidgetBackend.js

```diff
@@ -156,6 +156,13 @@ Meteor.methods({
 		};
 	},
 
+	closeConversation(roomId) {
+		if (RocketChat.authz.hasAllPermission(Meteor.userId(), 'archive-room', roomId)) { return false ; }
+		const room = RocketChat.models.Rooms.findOneById(roomId);
+		if (room.archived) { return false; }
```

Comment:
```suggestion
		return !room.archived;
```

Reply:
I consider the origin version more verbose